### ISSUE TITLE
Section 3: Scientific Deliverables

### ISSUE THREAD

**Issue by plasky** [plasky/OzHF#4]
Add issues about Section 3 here

**Comment by ilyamandel** [comment]
- [x] "where M, R, and k_2 are the mass..." -- m (lower case) was defined as the mass just a few lines earlier
- [x] "Large values of \Lambda imply soft equations of state, corresponding to small, compact neutron stars. Small values of Lambda imply stiff equations of state..." -- that's backwards, **small** Lambda correspond to compact NSs that are not significantly deformed by tides :)
- [x] "the **expected** signal-to-noise ratio" (just above Eq. 4; the actual SNR is a function of data, not just waveform)
- [x] "The most sensitive region of an EMO is designed to be sensitive to the physics of this late inspiral phase." -> "The EMO is designed to be optimally sensitive to the physics of this late inspiral phase."
- [x] "In Fig. 3, we show a reconstruction of the mass-radius relation using the loudest five neutron star mergers from the population." -- doesn't match the figure caption, which talks about 40 events.  Also, not clear what "loudest 5" (or 40) mean -- what merger rate and run duration are assumed?  Or is it 40 above detection threshold (which is what here?) from which the loudest 5 are chosen?  (Out of curiosity, what are the actual SNRs of these 5 loudest events?)
- [x] "[e.g., 67, and references therein]**.**" (missing period)
- [x] "For signals with matched filter signal-to-noise ratio >= 5" -- does that refer to the entire signal, or just the high-frequency portion?  If the former, why is this a reasonable choice (isn't this far too low for detection)?  If the latter, how is it defined?
- [x] "signal reconstruction losses track of phase" -> "signal reconstruction **loses** track of **the** phase"
- [x] "tracks the phase throughout the >= 10 ms." -> "tracks the phase for >= 10 ms."
- [x] Figure 4 caption: "The purple curve" -> "The black curve"?  
- [x] "whereas the reconstruction with EMO correctly reconstruct the gravitational-wave phase throughout the signal duration." -- grammar & wording
- [x] Figure 4: the top panel shows that the 90% credible interval is far too narrow for the signal; why should a reader trust the credible intervals produced by this algorithm?
- [ ] Figure 4: why is the injected signal so "spiky"?  Is it just under-sampled?  
- [x] Figure 5: yr^{-1} (make superscript) in ordinate caption
- [x] Figure 5 caption: "post-mergers signals" (grammar)
- [x] Figure 5: Why is the expected number of detections a non-monotonic function of peak frequency even though the noise curve is monotonic in this frequency range?  Perhaps this would be clear if the reader was told how the mock signals are generated.

**Comment by Nangush** [comment]
**Clarify Figure 4**

- [x] It is unclear that black represents the injected signal, and red the reconstructed signal. The use of red in figure 2 adds to this confusion. Can be fixed by adding a legend, and clarifying in the caption.

- [x] I would also recommend changing the injected signal's linestyle to dashed, or dotted.

**Comment by nikhil-sarin** [comment]
- [x] I have a concern with the way the science is discussed. I know the authors know about these things and maybe the science is expressed in this way because the alternative detracts from the message but I will write my concern here nonetheless.  

 
Focussing specifically on the matter information without commenting on the importance of robustly knowing the fate of the post-merger remnant is making the case for an EMO weaker than it really should be for me. In [1], the authors find that ~10 measurements of a joint EM + cold BNS event i.e more GW170817's is more accurate/comparable than the information from the ~40 candidates with an EMO/A+ network which could make a reader ask why build an EMO when it costs less to build an all-sky optical telescope (might have got the numbers wrong but from memory, GOTO is much cheaper than the costs previously discussed regarding OzHF, similarly Evryscope-like detector costs significantly less too). I am not too certain off the top of my head on what numbers of joint detections these detectors give you but that should be easy to check if required. We may end up in a position where there are ~10 joint EM/BNS by the time an EMO is operational, in which case a reader might ask how useful the cold matter constraints end up being. 

The obvious counter to this point which I feel presently appears to be lacking is that having an EMO provides a significantly less subjective interpretation into the nature of the post-merger remnant, this is **much less subjective than EM interpretations**. This is an important point. The paper as currently written appears to dive straight into the matter implications without emphasising that an **EMO is one of the better ways of inferring the nature of the post-merger remnant**. Otherwise, If you take [1] at face-value and believe EM interpretations are completely robust then you don't need an EMO to gain information about the cold EOS. You get it from joint observations. We know there are problems with this because EM interpretations are uncertain, but the paper does not make this point. 

In reality, an EMO is complementary and alleviates the problem in the analysis in [1] which is the uncertainty in the inferred fate given the current theoretical understanding of EM models. It makes the joint detections more informative! This should be emphasised IMO. 

- [x] Furthermore, Inferring the nature post-merger remnant can then be led back into answering important questions about kilonovae and GRBs; such as the impact of the neutrino radiation on r-process elements produced, the launching mechanism for GRB jets. The delay-time between jet-launching and production of prompt emission, several more things. 
  
[1] https://ui.adsabs.harvard.edu/abs/2019ApJ...880L..15M/abstract

**Comment by ddobie** [comment]
- [x] Figure 2 bottom panel - legend should include label for red line so it can be understood without reading the caption

- [x] Figure 4 - caption mentions a purple curve, but the plot shows curves in red and black

- [x] last sentence - "totally surprising" -> "unexpected"

**Comment by jade-powell** [comment]
- [x] Fig 2 - I would include the red signal in the legend. 

- [x] Fig. 2 still has OzHF in legend. 

- [x] Line 380 - All through the paper you switch between equation of state and EOS and never define EOS. 

- [x] Line 389 - I don't understand what is a fluffy neutron star. 

- [x] Line 403 - You say where A+ is located, but not EMO. 

- [x] "The inclusion of an EMO provides an approximately 50% improvement in the accuracy with which the equation of state can be measured". I wouldn't think that from looking at Figure 3. A+ and EMO looks better than A+ alone, but not loads better. 

- [x] With the BayesWave reconstruction, why should I care about having better phase information about the signal?

- [x] Fig 5 still has OzHF in the caption. 

- [x] I think the increased detection rate is one of the best results in the deliverables section and I would highlight it more in the abstract and intro.

**Comment by A-Graham** [comment]
- [x] line 512: "_matter eﬀects_" ?? 

- [x] lines 514 & 515:  do you want to address these two points in more detail, explaining how it will work?

- [x] It may be good to explain why NS-BH binaries have not been considered.

- [x] Section C leaves me feeling that EMO may only be for studying NS-NS coalescence and mergers.  If so, it seems like a rather specific/limited observatory.  Can this feeling be erased, perhaps by expanding on the above items.  For example, should there be separate subsections for what is currently line 514 and line 515?

**Comment by Chichisheep** [comment]
My naive two cents here:

- [x] In Fig 2. "Also shown in red is the characteristic gravitational-wave strain hc for a typical binary neutron star inspiral, merger, and post-merger at 40 Mpc." This is in conflict with the statement later at line 450: "The precise signal morphology of neutron star post- merger gravitational waves remains unknown."  maybe put some explanation/citation of obtaining this hc, from numerical relativity ? 

- [x] Sec. III A is trying to demonstrate that the addition of EMO can help measure EOS better. But the rendering of the main result, Fig.3, looks not that strong/convincing.  Green area has a similar size to the grey area, not shrinked a lot. The text mentioned it's reduced by 50%. maybe add a few more lines on how important is this 50% ? How costly to achieve the 50% if using other facilities ?

- [ ] Sec. III B mainly touches on the detection and reconstruction aspects, instead of "physics" as the III.B title suggests. Have another sub-section on the physics discussion or modify the title a bit?

- [x] Sec. III B, line 495 "Put another way, one would have to wait potentially many tens of years for a first post-merger detection without an EMO." Marice claims a detection of post-merger and I am inclined to believe that after his visit here.

**Comment by plasky** [comment]
>     * Figure 5: Why is the expected number of detections a non-monotonic function of peak frequency even though the noise curve is monotonic in this frequency range?  Perhaps this would be clear if the reader was told how the mock signals are generated.

@ilyamandel: it's because these are numerical-relativity simulations, so there's a lot more going on in the simulations than just the EOS that gives the peak frequency.  But I completely take your point that we need to describe this better in the text!

**Comment by plasky** [comment]
@ilyamandel, @jade-powell @Chichisheep: Thank you heaps for all your comments!  You've all made a similar comment about Fig. 3, that the improved sensitivity in the mass-radius diagram is not overwhelming, and the text is hard to follow.  You are all correct, and importantly, there's a mistake that we're fixing now.  

In short, we followed the procedure outlined in Fransisco's paper on stacking BNS posteriors to get EOS measurements.  In that paper, we found that almost all of the EOS information from the first 40 measurements is going to only come from about 5 or 6 of the loudest of those mergers.  So that's what we (naively) did here.  Unfortunately, that's not right because the addition of the ExMO means many of the slightly less loud signals will still significantly contribute to the EOS, where they won't for the network without ExMO.  This means we were doing ourselves a disservice!!  Mea Culpa!!!  

Fransisco is heroically working at getting new numbers in the eleventh hour.  It's not trivial, because our method for calculation is prohibitively expensive for lots of signals, so we've developed a new way of doing this, which is now coded up and running as we speak. 

TL;DR, I'm going to tick off your issues, because the plot is going to change anyway.  When it's updated, I'll post here the result!  Sorry for all the confusion.

**Comment by plasky** [comment]
>     * [ ]  Sec. III B, line 495 "Put another way, one would have to wait potentially many tens of years for a first post-merger detection without an EMO." Marice claims a detection of post-merger and I am inclined to believe that after his visit here.

Thanks again for the comments, @Chichisheep.  I assume in the above you're talking about van Putten and Della Valle (2018).  This work is very-much disputed, and I personally don't think this the ExMO paper is the place to go against conventional wisdom. 

To be more specific, this work by LIGO/Virgo members (arxiv:1812.06724) shows rigorously that the signal morphology found by vP-DV is inconsistent with emission from a neutron star.  As part of the 170817 post-merger papers from the LVC, many other sophisticated modeled and unmodeled searches were run on the data looking for this signal, but none were able to find anything.  

In addition, there is an entire body of numerical-relativity works going back 15 years that show the expected signal morphology of merger remnants that are in stark disagreement with that found by vP-DV.  The gravitational-wave amplitudes of these predictions, also supported by back-of-the-envelope energy-budget arguments, are what we are using when we talk about event rates with and without an EMO.  There is a wealth of literature backing up those predictions.

I am more than happy to chat about this more: it is a topic I have done a lot of work on, and I would love nothing more than to finally detect a post-merger remnant.

**Comment by Chichisheep** [comment]
> > ```
> > * [ ]  Sec. III B, line 495 "Put another way, one would have to wait potentially many tens of years for a first post-merger detection without an EMO." Marice claims a detection of post-merger and I am inclined to believe that after his visit here.
> > ```
> 
> Thanks again for the comments, @Chichisheep. I assume in the above you're talking about van Putten and Della Valle (2018). This work is very-much disputed, and I personally don't think this the ExMO paper is the place to go against conventional wisdom.
> 
> To be more specific, this work by LIGO/Virgo members (arxiv:1812.06724) shows rigorously that the signal morphology found by vP-DV is inconsistent with emission from a neutron star. As part of the 170817 post-merger papers from the LVC, many other sophisticated modeled and unmodeled searches were run on the data looking for this signal, but none were able to find anything.
> 
> In addition, there is an entire body of numerical-relativity works going back 15 years that show the expected signal morphology of merger remnants that are in stark disagreement with that found by vP-DV. The gravitational-wave amplitudes of these predictions, also supported by back-of-the-envelope energy-budget arguments, are what we are using when we talk about event rates with and without an EMO. There is a wealth of literature backing up those predictions.
> 
> I am more than happy to chat about this more: it is a topic I have done a lot of work on, and I would love nothing more than to finally detect a post-merger remnant.

Thanks for the reply. I am very interested in this topic. I don't have much background of the expectations for post-mergers and appreciate your comments on that. Hopefully can discuss more about that given chance.

I looked at this problem purely from the point of view of data and algorithms. All I can say is that there is nothing wrong with his method, and there is definitely something post merger. I was trying to reproduce his result but paused due to my leave. This may give me more motivation to finish it. Nevertheless, it requires some computation as searching for a large space of chirplets that may take some time.
